### DISCUSSION TITLE
Coarseplay Without Automation?

### DISCUSSION THREAD

**Discussion by uf2-farmsim** [Courseplay/Courseplay_FS22#2833] [Ideas]
Coarseplay Without Automation?

Would it be possible to remove the self-driving portion of Coarseplay?  I ask because it would be nice to generate the paths on the realistic (realism) servers as you can in real life, but you would have to drive the tractor.  It would only lock into the path for the steering.

When on a map with many fields with odd angles, the guidance mod does not work well for headlands.  It would be nice to be able to generate the field paths as you would with coarseplay, but it does not drive the machinery.  No automation.  You basically would use the coarse to lock into the path and only automate the steering.  All other operations would be performed by the driver.

This would make it legit to be useable on realism servers.  It would basically be used for steering only.

**Comment by Tensuko** [comment]
That won't work as many stuff relies on AI mechanics, such as turns.
Won't happen, sry.

If you want to work real, use GPS by Wopster + Headland Management Mod.
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=228522&title=fs2022
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=228759&title=fs2022
